app/templates/error.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
You can use template inheretance.  All common staff are put to base tamplate. (Optional)